src/main/java/com/xiaomi/infra/pegasus/client/PException.java

```diff
@@ -33,20 +34,24 @@ public PException(Throwable cause) {
     super(versionPrefix + cause.toString(), cause);
   }
 
-  static PException threadInterrupted(String tableName, InterruptedException e) {
+  static PException threadInterrupted(
+      String metaList, String tableName, Request request, InterruptedException e) {
     return new PException(
         new ReplicationException(
             error_code.error_types.ERR_THREAD_INTERRUPTED,
-            String.format("[table=%s] Thread was interrupted: %s", tableName, e.getMessage())));
+            String.format(
+                "[metaServer=%s, table=%s, request=%s] Thread was interrupted: %s",
+                metaList, tableName, request.toString(), e.getMessage())));
```

Comment:
`threadInterrupted` doesn't need such a verbose error message, because `InterruptedException` has nothing to do with Pegasus service. Revert it to the previous version.